tools/check-license.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+import re
+import sys
+
+LICENSE = re.compile(
+    r'((#|//|\*) Copyright .*\n'
+    r')+\s?\2\n'
+    r'\s?\2 Licensed under the Apache License, Version 2.0 \(the "License"\);\n'
+    r'\s?\2 you may not use this file except in compliance with the License.\n'
+    r'\s?\2 You may obtain a copy of the License at\n'
+    r'\s?\2\n'
+    r'\s?\2     http://www.apache.org/licenses/LICENSE-2.0\n'
+    r'\s?\2\n'
+    r'\s?\2 Unless required by applicable law or agreed to in writing, software\n'
+    r'\s?\2 distributed under the License is distributed on an "AS IS" BASIS\n'
+    r'\s?\2 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n'
+    r'\s?\2 See the License for the specific language governing permissions and\n'
+    r'\s?\2 limitations under the License.\n'
+)
+
+INCLUDE_DIRS = [
+    'src',
+    'tools',
+]
+
+EXCLUDE_DIRS = [
+]
+
+EXTENSIONS = [
+    '.c',
+    '.cmake',
+    '.cpp',
+    '.h',
+    '.js',
+    '.py',
+    '.S',
+    '.sh',
+    '.tcl',
```

Comment:
I guess it makes sense to keep these around to catch future additions?

Reply:
I'm also voting for the shorter list. For now, something like the following might be needed: .sh, .py, .js, .ts?